swagger.yaml

```diff
@@ -199,20 +296,29 @@ components:
         password:
           type: string
           example: Password1@
-    CompileOptions:
+    Project:
       type: object
       properties:
-        entrypoint:
+        id:
+          type: integer
+          format: int64
+          example: 1
+        owner:
+          type: integer
+          format: int64
+        main_document_id:
+          type: integer
+          format: int64
+          example: 1
+        name:
           type: string
-          nullable: true
-          example: main.tex
-        force:
-          type: boolean
-          example: false
-        text:
+          example: sample_project
```

Comment:
After some discussion we decided that some other stuff are important, so it should include:
* `project_id` - project ID
* `name` - project name
* `owner` - owner name
* `created_at` - creation timestamp
* `last_modified` - last modification timestamp